tensorflow/python/ops/math_ops.py

```diff
@@ -1484,7 +1529,7 @@ def range(start, limit=None, delta=1, dtype=None, name="range"):  # pylint: disa
   `range(n) = range(0, n)`.
 
   For example:
-
+  
```

Comment:
![LAEO9bhS4sW](https://user-images.githubusercontent.com/323199/72914914-4aa16780-3cf4-11ea-92da-049b39d70158.png)

Reply:
Trailing whitespace is still here